components/ecosystem/TestTable.vue

```diff
@@ -38,26 +57,41 @@ import { TableRowElement } from '~/components/ui/AppDataTable.vue'
 
 @Component
 export default class TestTable extends Vue {
-  @Prop({ type: Array, default: () => [['', '', '']] }) filteredData!: Object[];
+  @Prop({ type: Array, default: () => [['', '', '']] }) filteredData!: Object[]
+
+  testTypeTooltip = {
+    development: 'This test type indicates the ecosystem tests were run for this package using the latest development version of Qiskit',
+    stable: 'This test type indicates the ecosystem tests were run for this package using the latest stable version of Qiskit',
+    standard: 'This test type means the ecosystem tests were run for this package using the Qiskit version specified in the package requirements',
+    'last passing version': 'This test type means the results in this row show the latest version of Qiskit for which the ecosystem tests pass for this package'
+  }
 
   tableData = this.dataPerRow(this.filteredData);
 
   dataPerRow (filteredData: Object[]): TableRowElement[][] {
-    return filteredData.map(({ passed, testType, terraVersion }: any) => [
-      {
-        component: passed,
-        styles: passed ? '#42be65' : '#da1e28',
-        data: ''
-      },
-      {
-        component: 'span',
-        data: testType
-      },
-      {
-        component: 'span',
-        data: terraVersion
-      }
-    ])
+    return filteredData.map(
+      ({ passed, testType, packageVersion, logsLink, packageName }: any) => [
+        {
+          component: passed,
+          styles: passed ? '#42be65' : '#da1e28',
```

Comment:
non-blocking comment: it would've been nice to use classes and Carbon color tokens...